components/automate-chef-io/content/docs/migrate.md

```diff
@@ -110,7 +110,7 @@ To configure Chef Automate 1 backups, see the [Chef Automate 1 Documentation](ht
 Chef Automate 2 includes significant architectural and technical improvements to the core product platform.
 For the initial releases of this new major version, we do not yet support a handful of capabilities that were present in Chef Automate 1.
 If you rely on any of the capabilities listed below, we recommend you continue to use Chef Automate 1.
-Chef continues support of Chef Automate 1 while we develop upgrade paths for these capabilities on the new platform.
+However, note that Chef Automate 1 reached end-of-life on December 31, 2019.
```

Comment:
```suggestion
Note that Chef Automate 1 reached end-of-life on December 31, 2019.
```

Reply:
A couple of issues:
1) The EOL needs to be at the top of the paragraph.
2) There is only one Chef Automate in terms of product and marketing; Chef Automate 1 & 2 are useful fictions. We should clarify something along the lines of "The earlier version of Chef Automate, also referred to as Chef Automate 1 and the current version of Chef Automate..." etc.,

Reply:
@kagarmoe How about you go ahead and update the text? I don't mind at all. 😁